### PR TITLE
[5.3] Relocated clone notification on ChannelManager

### PR DESCRIPTION
**Pull request by socieboy**
It wasn't doing effect where it was before!

### CODE REVIEW

File: src/Illuminate/Notifications/ChannelManager.php [hunk 1]
```diff
@@ -57,7 +57,6 @@ public function sendNow($notifiables, $notification)
         }
 
         foreach ($notifiables as $notifiable) {
-            $notification = clone $notification;
```
Comment (GrahamCampbell) [inline src/Illuminate/Notifications/ChannelManager.php]: Please remove the extra newline.

**Comment by GrahamCampbell** [comment]
I don't think this is correct?

File: src/Illuminate/Notifications/ChannelManager.php [hunk 2]
```diff
@@ -57,7 +57,6 @@ public function sendNow($notifiables, $notification)
...
 
             $notification->id = (string) Uuid::uuid4();
```
Comment (GrahamCampbell) [inline src/Illuminate/Notifications/ChannelManager.php]: The line under this one's behaviour is now changed?

Comment (socieboy) [inline src/Illuminate/Notifications/ChannelManager.php]: Did you test it?

**Comment by GrahamCampbell** [comment]
I don't understand this. The cloning was intentional to prevent the uuid modification from screwing with stuff.

**Comment by socieboy** [comment]
Well we can leave it before to assign the ID, but it has to be cloned on the send method also.

I report that issue.
https://github.com/laravel/framework/issues/14883

**Comment by taylorotwell** [comment]
Please provide an example notification class we can use to test this and prove your problem.

**Comment by taylorotwell** [comment]
But, no, this PR definitely does not look correct. We need a sample notification to recreate your problem.

**Comment by taylorotwell** [comment]
Also would like to know your 5.3 version.

**Comment by socieboy** [comment]
I just tested again with Laravel v5.3.0-RC1, I got the same error without add the clone on the send method.

I'm creating a example repo to show you.

**Comment by socieboy** [comment]
check this example @taylorotwell 

https://github.com/socieboy/notifications-test

**Comment by socieboy** [comment]
Well, the it doesn't need to be passed on the send method to every driver, instead of that it could be only cloned for the broadcast driver on the send method, since is here where the problem happen.

```
// Illuminate\Notifications\Channels\BroadcastChannel

public function send($notifiable, Notification $notification)
    {
        $notification = clone $notification;
        $this->events->fire(new BroadcastNotificationCreated(
            $notifiable, $notification, $this->getData($notifiable, $notification)
        ));
    }
```

**Comment by taylorotwell** [comment]
Pretty sure I have it fixed. Update your dependencies and try again. Note that you **must** remove the "prefer-stable" from your composer.json to get the latest laravel/framework.

**Comment by socieboy** [comment]
I notice your fix it, but i think it's not necessary to clone it for each channel twice, the problem happened only on the broadcast channel.

**Comment by taylorotwell** [comment]
So did it fix it or not?

**Comment by socieboy** [comment]
it does, but i don't know why you need to clone it for the others channels if it's only necessary for the broadcast. but it's fine, i just said! thanks!

**Pull request closed without merging**